erpnext/utils.py

```diff
@@ -52,4 +53,75 @@ def create_authorization_request(dt, dn, contact_email, contact_name):
 	new_authorization_request.linked_docname = dn
 	new_authorization_request.authorizer_email = contact_email
 	new_authorization_request.authorizer_name = contact_name
-	new_authorization_request.save()
\ No newline at end of file
+	new_authorization_request.save()
+
+@frappe.whitelist(allow_guest=True)
```

Comment:
why does this function has `allow_guest=True`, the login_as button is added on the user doctype and the user is already logged in at this point.

Reply:
any update on this?

Reply:
hey @sahil28297  i messaged @DeveloperMetal  regarding this if i get reply will update or simply i will remove this allow_guest by evening